### ISSUE TITLE
gperf crosscompiling fails (bash: ar: command not found)

### ISSUE THREAD

**Issue by whohoho** [NixOS/nixpkgs#50749]
## Issue description

i686-unknown-linux-gnu-g++ -g -O2 -I. -c ./hash.cc
  GEN      lib/stdlib.h
source root is autoconf-2.69
  GEN      lib/string.h
rm -f libgp.a
ar rc libgp.a getopt.o getopt1.o getline.o hash.o
/nix/store/r47p5pzx52m3n34vdgqpk5rvqgm0m24m-bash-4.4-p23/bin/bash: ar: command not found
make[1]: *** [Makefile:72: libgp.a] Error 127
make[1]: Leaving directory '/build/gperf-3.0.4/lib'
make: *** [Makefile:33: all] Error 2
setting SOURCE_DATE_EPOCH to timestamp 1335323187 of file autoconf-2.69/ChangeLog
patching sources
configuring
  GEN      lib/sys/stat.h
builder for '/nix/store/clijzzs1s4wb0mx13vcchcf2sq8wlcif-gperf-3.0.4-i686-unknown-linux-gnu.drv' failed with exit code 2

### Steps to reproduce

in nixops:

nixpkgs.crossSystem.system = "i686-linux";

## Technical details

nixos-unstable channel
 - system: `"x86_64-linux"`
 - host os: `Linux 4.18.11, NixOS, 19.03pre-git (Koi)`

**Comment by eburimu** [comment]
```
  9993 [void@fermat:~/pi]$ sh ./gperf.sh
  9994 /nix/store/65cbhqmhvh9a9ngcyi1jacj1shr8nqg0-gperf-3.1-armv6l-unknown-linux-gnueabihf
  9995 
  9996 [void@fermat:~/pi]$ cat ./gperf.sh
  9997 #!/bin/sh
  9998 nix-build -I nixpkgs=/home/void/nixpkgs.orig -E 'let t1 = import <nixpkgs> {}; in let pkgs = import <nixpkgs> { crossSystem = t1.lib.\
       systems.examples.raspberryPi // { system = "armv6l-linux"; }; }; in pkgs.gperf '
```

Maybe you should upgrade?

**Comment by sikmir** [comment]
Fixed by #51210
